prometheus_metrics/prometheus_metrics.py

```diff
@@ -1,4 +1,6 @@
 from prometheus_client import Counter, generate_latest
+from prometheus_client import CollectorRegistry, multiprocess
```

Comment:
Just a nitpick here:
In Python it's generally preferred to group things rather than repeat yourself. So, when import many objects from a package and it doesn't fit into a single line, it's preferred to group it like this:

```suggestion
from prometheus_client import (Counter, generate_latest,
    CollectorRegistry, multiprocess)
```

See [PEP 328](https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses) for more details. :slightly_smiling_face:

Reply:
will do, thanks!